vtysh/vtysh.c

```diff
@@ -4721,6 +4721,22 @@ static int vtysh_connect(struct vtysh_client *vclient)
 		close(sock);
 		return -1;
 	}
+	/* Setting the RCV buffer size to 16MB to increase the buffer size
```

Comment:
please don't embed the value in the comment

Reply:
Will do